util/bazelisk.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Also, not to ask the obvious, but why shell instead of python? To make it easier to exec into bazelisk?

Reply:
Lowest common denominator.